lib/stream_reader/librato_reporter.rb

```diff
@@ -17,6 +18,13 @@ def run!
     end
 
     private
+    
+    def config_present?
+      !(ENV['LIBRATO_USER'].nil?) &&
+        !(ENV['LIBRATO_USER'].empty?) &&
+        !(ENV['LIBRATO_TOKEN'].nil? &&
```

Comment:
D'oh. That's what I get for using Github.com to make a PR without syntax highlighting.